omar-abdelgawad/image2image
# Changing proj conf

## 📢 Describe your changes

- 📌 **changes Summary**
  - Removed old configuration like `setup.py`, `setup.cfg`, `requirements.txt` and others for only `pyproject.toml`
  - Replaced pylint with Ruff.
  - Formatted and Linted entire codebase.
- ⚠️ **Breaking changes**
  - None
- 🐞 **Bug Fixes**:
    - None

## 🧪 How to test this Pull Request

1. Delete current virtual environment and reinstall the package with `pip install -e ".[dev]"` in a new environment.
2. Try pytest using `pytest` should take about 12 seconds.
3. Try tox using `tox` should take about 288 seconds.


## 🗒️ Notes for future improvements


## ✅Checklist before requesting a review
